ivy/functional/frontends/torch/reduction_ops.py

```diff
@@ -51,3 +52,20 @@ def prod(input, dim=None, keepdim=False, *, dtype=None, out=None):
 
 def var(input, dim, unbiased, keepdim=False, *, out=None):
     return ivy.var(input, axis=dim, correction=int(unbiased), keepdims=keepdim, out=out)
+
+
+def min(input, dim=None, keepdim=False, *, out=None):
+    if dim is None:
+        ret = ivy.min(input, axis=dim, keepdims=keepdim, out=out)
```

Comment:
we can simply `return` here, and in fact we can do this inside each of the conditional blocks. No need to wait until the end of the function to return things. This in general makes it easier to read IMO, you know it's no longer necessary to continue following the logic further down the page once you see the `return` command in a certain conditional block.